packages/pg-meta/src/pg-meta-tables.ts

```diff
@@ -107,14 +107,14 @@ function list<T extends boolean | undefined = true>(
 
 function retrieve(identifier: TableIdentifier): {
   sql: string
-  zod: z.ZodType<PGTable | undefined>
+  zod: z.ZodType<TableWithColumns>
 } {
   let whereClause = getIdentifierWhereClause(identifier)
 
   const sql = `${generateEnrichedTablesSql({ includeColumns: true })} where ${whereClause};`
   return {
     sql,
-    zod: pgTableZod.optional(),
+    zod: pgTableZod,
```

Comment:
```suggestion
    zod: pgTableZod.optional(),
```

this is still optional because result can be empty array right?